src/main/java/Events.java

```diff
@@ -0,0 +1,21 @@
+public class Events extends Task {
+    protected String eventAt;
```

Comment:
Strange that most of your classes use protected variables.
Is there a reason why they are not private instead?